tests/security/scripts/authz_tcp_traffic.txt

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+# Export HOST, PORT and URL
+# $snippet export_gateway_url_1.sh syntax="bash"
+$ export INGRESS_HOST={{ .ingressHostCommand }}
+$ export INGRESS_PORT={{ .ingressPortCommand }}
+$ export GATEWAY_URL=$INGRESS_HOST:$INGRESS_PORT
+# $endsnippet
+
+# Step 1: Install v2 of the ratings workload with the bookinfo-ratings-v2 service account
+# $snippet bookinfo-ratings-v2.sh syntax="bash" outputis="text"
+$ kubectl apply -f @samples/bookinfo/platform/kube/bookinfo-ratings-v2.yaml@
+# $endsnippet
+
+# Step 2: Create the appropriate destination rules
+# $snippet create-destination-rule.sh syntax="bash" outputis="text"
+$ kubectl apply -f @samples/bookinfo/networking/destination-rule-all-mtls.yaml@
+# $endsnippet
+
+# wait for destination rules to propagate
+# $snippet wait_for_product_destination_rule_1.sh syntax="bash"
+$ istioctl experimental wait --for=distribution DestinationRule productpage.default --timeout=60s
+# $endsnippet
+
+# Step 3: update the reviews workload to only use the v2 of the ratings workload
+# $snippet update-reviews-v2.sh syntax="bash" outputis="text"
+$ kubectl apply -f @samples/bookinfo/networking/virtual-service-ratings-db.yaml@
+# $endsnippet
+
+# Step 4: Verify ratings is unavailable
```

Comment:
I'm not quite sure what the phrase "Verify ratings is unavailable" means--I can think of a couple of different things that it might mean. Can you rephrase slightly for clarity?

This question also also applies to step 8, below. ("Verify ratings is unavailable again")